utils/charts/chartmuseum.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
I don't think you need to bother yourself with hosting a chart registry, you can simply apply the chart from the local disk.

Reply:
Ah neat, didn't know. Regardless, its there now.